en/developers-guide/proof-of-reserve.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Ongoing proof of reserve validation for your financial entity.
+---
+
+# Proof of Reserve
+
+For a comprehensive illustration of utilizing **Orakl Network's Proof of Reserve**, refer to the 
+example repository of the [`data-feed-consumer`](https://github.com/Bisonai/data-feed-consumer). The on-chain implementation for Proof of Reserve mirrors that of the Aggregator on the Aggregator Proxy.
+
+## What is Proof of Reserve?
+
+The **Orakl Network's Proof of Reserve (PoR)** stands as a vital element in fostering trust and transparency within the financial ecosystem of 
+Orakl Network. Playing a pivotal role, Proof of Reserve is dedicated to the verification and validation of reserve holdings for financial 
+entities through a secure and auditable procedure.The Peg-Gold Proof of Reserve refers to a reserved quantity of non-fungible tokens (NFTs) issued specifically for PEG Gold.
+
+
+### Supported Proof of Reserve on Cypress
+
+<table><thead><tr><th width="157">PoR</th><th>Aggregator</th><th>AggregatorProxy</th><th>Heartbeat (m)</th></tr></thead><tbody>
+    <tr>
+        <td>PEG-POR</td>
+        <td><a href="https://www.klaytnfinder.io/account/0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1">0xb5e91e5CE0B8e6fc3029b4E9ce057675a2c96dd1</a></td>
+        <td><a href="https://www.klaytnfinder.io/account/0x9FbA23B10692cB3fa6Fea09834855ACc597BD180">0x9FbA23B10692cB3fa6Fea09834855ACc597BD180</a></td>
+        <td>60</td></tr></tr></tbody></table>
+
+### Supported Proof of Reserve on Baobab
```

Comment:
Please mention in here that the Baobab is using development API which does not have to return the same results as when used with production API used for Cypress deployment.